### DISCUSSION TITLE
Has anything changed between yesterday and today? no longer get 256x256 with 2060 6gb vram

### DISCUSSION THREAD

**Discussion by Appolonius001** [deforum-art/sd-webui-text2video#70] [Q&A] [answered]
yesterday i was able to render videos at 256x256 with only a 6gb vram card using --lowvram setting. Has something changed?
![image](https://user-images.githubusercontent.com/111724828/227734723-d53feaee-292e-42a7-9d70-ae894416795d.png)

**Comment by Appolonius001** [comment, marked as answer]
strange, i switched from --lowvram to --medvram and now i can do 256x256 again

**Reply by kabachuha** [reply]
Great! It's recommended to launch it with medvram too, since lowvram optimizations are weirder

The extension is in active development, so breaking changes may happen from time to time

**Reply by Appolonius001** [reply]
thats perfectly fine :D you are doing some Amazing work here thank you!
